### PR TITLE
HTTPS fix

### PR DESCRIPTION
**Pull request by cloveras**
Let's try to get everyone to live in the present, and use HTTPS.

### CODE REVIEW

File: docs/swagger.yaml [hunk 1]
```diff
@@ -1984,6 +1993,7 @@ components:
             In other words: This is the URL Vipps sends the Vipps user back to.
             URLs passed to Vipps must validate, see
             [URL validation](https://github.com/vippsas/vipps-ecom-api/blob/master/vipps-ecom-api.md#url-validation).
+            The URL must use HTTPS.
```
Comment (tht13) [inline docs/swagger.yaml:1996]: fallback doesn't require https
its only https if its a web redirect, but native switch like `ruter://` is allowed

File: vipps-ecom-api-faq.md [hunk 2]
```diff
@@ -884,6 +884,8 @@ for problems with the callbacks. The API Dashboard is under "Utvikler".
 If you need help solving a callback-related problem, please send us a
 complete HTTP request, and any other related details, so we can investigate.
 
+**Please note:** Cllback URLs _must_ use HTTPS.
```
Comment (tht13) [inline vipps-ecom-api-faq.md:887]:
```suggestion
**Please note:** Callback URLs _must_ use HTTPS.
```

File: vipps-ecom-api.md [hunk 3]
```diff
@@ -615,9 +616,9 @@ as the app-switch URL may be something like `vipps://`, which is not a valid URL
 The endpoints required by Vipps must be publicly available.
 
 URLs that start with `https://localhost` will be rejected. If you want to use
-localhost as fallback, please use `http://127.0.0.1`.
+localhost as fallback, please use `https://127.0.0.1`.
```
Comment (tht13) [inline vipps-ecom-api.md:619]: this doesn't work with the sentence (and https doesn't work on localhost) 😅 we'll allow http on localhost for testing

File: vipps-ecom-api.md [hunk 4]
```diff
@@ -674,6 +676,8 @@ confirmation on the payment request, the merchant _must_ call
 [GET:/ecomm/v2/payments/{orderId}/details](https://vippsas.github.io/vipps-ecom-api/#/Vipps_eCom_API/getPaymentDetailsUsingGET)
 to get the status of the payment.
 
+**Please note:** Cllback URLs _must_ use HTTPS.
```
Comment (tht13) [inline vipps-ecom-api.md]:
```suggestion
**Please note:** Callback URLs _must_ use HTTPS.
```

**Review by tht13 — CHANGES_REQUESTED** [review]

File: docs/swagger.yaml (hunk 1 above)
Comment (cloveras) [reply docs/swagger.yaml]: Thanks, added/fixed.

File: vipps-ecom-api-faq.md (hunk 2 above)
Comment (cloveras) [reply vipps-ecom-api-faq.md:887]: Alrteady done! 🤦‍♂️

File: vipps-ecom-api.md (hunk 3 above)
Comment (cloveras) [reply vipps-ecom-api.md:619]: Ach! Thanks.

File: vipps-ecom-api.md (hunk 4 above)
Comment (cloveras) [reply vipps-ecom-api.md]: Uh-oh

File: vipps-ecom-api-faq.md (hunk 2 above)
Comment (tht13) [reply vipps-ecom-api-faq.md:887]: emmm no its still here, you must have undone it

File: vipps-ecom-api.md [hunk 5]
```diff
@@ -395,6 +395,7 @@ If Vipps is installed, Vipps will automatically be opened.
    as this depends on user actions, network connectivity/speed, etc.
    Because of this, it is not possible to base an integration on a specific
    sequence of events.
+5. Callback and fallback URLs _must_ use HTTPS.
```
Comment (tht13) [inline vipps-ecom-api.md:398]: slight over precision on fallback here

File: vipps-ecom-api-faq.md (hunk 2 above)
Comment (cloveras) [reply vipps-ecom-api-faq.md]: Ugh, missed the FAQ file.

File: vipps-ecom-api.md (hunk 5 above)
Comment (cloveras) [reply vipps-ecom-api.md]: Even more overprecision now:
---
The `callbackPrefix` URL _must_ use HTTPS.
The `fallBack` URL must use either HTTPS or a custom URL scheme (`myapp://`).

**Review by tht13 — APPROVED** [review]

**Pull request merged**